### ISSUE TITLE
Validation error for objects in properties

### ISSUE THREAD

**Issue by wilzbach** [microservices/omg#286]
The following is a valid OMS spec:

```yaml
omg: 1
info:
  version: 1.0.0
  title: Your microservice
  description: Describe your microservice
  license:
    name: Apache 2.0
    url: https://opensource.org/licenses/Apache-2.0
lifecycle:
  startup:
    command: []
health:
  http:
    path: /health
    port: 8080
actions:
  message:
    http:
      path: /message
      method: post
      port: 8080
    arguments:
      name:
        type: object
        properties:
          message:
            type: string
        required: true
        in: requestBody
    output:
      contentType: application/json
      type: object
      properties:
        message:
          type: string
```

ERROR:

```
ℹ Validation failed with the following errors: 
  - .actions.message.arguments.name.properties.message.in is required
```

**Comment by StoryscriptHuman** [comment]
Thank you for the report!

**Comment by StoryscriptHuman** [comment]
Fixed in 81b8bc139fec7e9d32c5dfc70d866b19d8b54691
